cyphernodeconf_docker/templates/installer/docker/docker-compose.yaml

```diff
@@ -245,9 +245,8 @@ services:
 
   proxycron:
     image: cyphernode/proxycron:<%= proxycron_version %>
-    environment:
-      - "TX_CONF_URL=proxy:8888/executecallbacks"
-      - "OTS_URL=proxy:8888/ots_backoffice"
+    env_file:
+      - ./.env/proxycron.env
```

Comment:
(A) - This works : - .env/proxycron.env.  Without the ./ thing

(B) - Or we could write in the template :
    env_file:
      - <%= setupDir %>/.env/proxycron.env
 
 ==> env_file:
      - /Users/xxxxx/werk/features/env-proxycron/cyphernode/dist/.env/proxycron.env
      
 You get the full path (B) in the docker-compose.yaml instead of relative to where the file is with (A)

Reply:
In docker-compose.yaml, does `.` always correspond to the dir where docker-compose.yaml file is?  For example:

```
docker stack deploy -c path/to/docker-compose.yaml stackname
```

In this case, is `.` the place where the `docker stack deploy` is executed or does it correspond to `path/to/`?

If `.` is always the dir where the yaml file is, then it's fine to use it.  If not, we should use a full path instead.  The same question for if we use `$PWD` in the yaml file...